runtime/environment.go

```diff
@@ -1127,41 +1096,44 @@ func (e *interpreterEnvironment) loadContract(
 	invocationRange ast.Range,
 ) *interpreter.CompositeValue {
 
-	switch compositeType.Location {
-	case stdlib.CryptoCheckerLocation:
-		contract, err := stdlib.NewCryptoContract(
-			inter,
-			constructorGenerator(common.ZeroAddress),
+	var contractValue interpreter.Value
+
+	location := compositeType.Location
+	switch location := location.(type) {
+	case common.IdentifierLocation:
+		// Identifier locations are used for built-in contracts,
+		// and are not stored in storage
+
+		constructorGenerator := constructorGenerator(common.ZeroAddress)
+		var err error
+		contractValue, err = inter.InvokeFunctionValue(
```

Comment:
This is not ideal: Ideally we would like to load the contract value that was deployed on-chain. That could be implemented, but then the defensive checks (e.g. member access) prevent using the contract value that has a composite type with address location, as a contract value for the imported identifier location.

For now, just re-instantiate the contract on each import. For the `Crypto` contract in particular that does not matter anyway, as it does not have a constructor.